packages/itmat-interface/test/serverTests/_loginHelper.ts

```diff
@@ -2,9 +2,12 @@ import { print } from 'graphql';
 import * as itmatCommons from 'itmat-commons';
 import * as mfa from '../../src/utils/mfa';
 import { SuperTest, Test } from 'supertest';
-
+import config from '../../config/config.sample.json';
 const { LOGIN, LOGOUT } = itmatCommons.GQLRequests;
 
+/* Choose API version tp use */
+const apiPath = '/api/'.concat(config.apiVersions[config.apiVersions.length - 1]);
```

Comment:
When testing we still need to verify the previous versions are still passing there tests, otherwise this might lead us into error when older software uses them and they break everything.